lib/Support/GitSHA1.cpp.in

```diff
@@ -0,0 +1,2 @@
+#define GIT_SHA1 "@GIT_SHA1@"
+extern const char g_GIT_SHA1[] = GIT_SHA1;
```

Comment:
I don't think `extern` is needed here.

The final version that is printed is
```
  SeaHorn version 0.1.0-rc3
  GitSHA1 d8de68301001a7eb375aed299e23a5f7f95da41c
```
It would be better if it would print
```
SeaHorn version 0.1.0-rc3-d8de6830
```

While `sha1` are large, usually a small subset is shown. Not sure whether we have to cut it or there is some other git command to get the short value. I would check git commands first.

Reply:
The extern is needed (at least in my environment). The reason is in here https://stackoverflow.com/questions/1358400/what-is-external-linkage-and-internal-linkage/1358796#1358796

Reply:
SHA1 is trimmed in https://github.com/seahorn/seahorn/pull/183/commits/6f0765a3378a64a6eda1b74516dc22957ae20eb5